tral/tral_configuration/config.ini

```diff
@@ -8,14 +8,14 @@ sequence_type = AA
         DNA = Phobos, TRED, T-REKS, TRF, XSTREAM
     [[repeat_detector_path]]
         # If the executable is in the system path, supply its name. Otherwise, supply the full path to the executable. Details are explained in TRAL's online docs.
-        PHOBOS = phobos
+        PHOBOS = phobos_*
```

Comment:
Did you test this? I'd be surprised if the glob * got expanded.

Reply:
I thought it worked but now I changed it to a symlink.